src/test/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporterTest.java

```diff
@@ -177,6 +178,19 @@ public void shouldCorrectlyRecordResponseErrors() throws IOException {
         assertGenericMetric("keycloak_response_errors", 1, tuple("code", "500"), tuple("method", "POST"));
     }
 
+    @Test
+    public void shouldCorrectlyRecordSessions() throws IOException {
+
+        Map<String,Long> map1 = Map.of("account", new Long(10), "admin-cli", new Long(0));
+        Map<String,Long> map2 = Map.of("account", new Long(0), "admin-cli", new Long(10));
```

Comment:
- sourceCompatibility = 1.8(build.gradle:20)
- `Map.of()` is supported since Java 9

Reply:
I rewrite it with : 
```
        Map<String,Long> map1 = new HashMap<String, Long>() {{
            put("account", new Long(10));
            put("admin-cli", new Long(0));
        }};
```

Is it better ?